third_party/bazel/tf/tf_configure.bzl

```diff
@@ -44,15 +44,15 @@ def _tf_configure_impl(repository_ctx):
         "-DTF_MAJOR={}".format(tf_major),
         "-DTF_MINOR={}".format(tf_minor),
     ]
-    if tf_is_pai:
+    if tf_is_pai == "True":
```

Comment:
looks weird ... why not making `tf_is_pai` a boolean variable, and use `str(tf_is_pai)` for templating.

Reply:
`tf_is_pai` is acquired from env as string use `get_env_bool_value_str` func, maybe using `if bool(tf_is_pai)` here is ok.

Reply:
done